ui/src/config/section/network.js

```diff
@@ -38,7 +38,7 @@ export default {
         return fields
       },
       details: () => {
-        var fields = ['name', 'id', 'description', 'type', 'traffictype', 'vpcid', 'vlan', 'broadcasturi', 'cidr', 'ip6cidr', 'netmask', 'gateway', 'aclname', 'ispersistent', 'restartrequired', 'reservediprange', 'redundantrouter', 'networkdomain', 'zonename', 'account', 'domain']
```

Comment:
@Pearl1594 
would it be good to highlight the egress policy or the behavior (allow/deny) of egress rules in "Egress rules" tab ?
e.g.  if egress policy is true,  display a message like "The traffic matching the following rules will be denied"

Reply:
Sure @weizhouapache , I can make the necessary changes to highlight was is implied when egress policy is true / false. Thanks